src/test/java/ru/training/at/hw2/ex2/ExerciseSecond.java

```diff
@@ -0,0 +1,129 @@
+package ru.training.at.hw2.ex2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import ru.training.at.hw2.TestBaseClass;
+
+public class ExerciseSecond extends TestBaseClass {
+
+    @BeforeTest
+    @Override
+    public void beforeTest() {
+        super.beforeTest();
+    }
+
+    @Test
+    public void homePageTest() {
+
+        //1.Open test site by URL
+        webDriver.get("https://jdi-testing.github.io/jdi-light/index.html ");
+
+        //2.Assert Browser title
+        (new WebDriverWait(webDriver, 10))
+                .until(ExpectedConditions.titleContains(webDriver.getTitle()));
+        String actualTitle = webDriver.getTitle();
+        String expectedTitle = "Home Page";
+        Assert.assertEquals(actualTitle, expectedTitle);
+
+        //3.Perform login
+        WebElement loginButton = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.elementToBeClickable(webDriver.findElement(By.id("user-icon")))));
+        loginButton.click();
+        WebElement username = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By.id("name"))));
+        username.sendKeys("Roman");
+        WebElement password = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By.id("password"))));
+        password.sendKeys("Jdi1234");
+        WebElement enterButton = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@id='login-button']"))));
+        enterButton.click();
```

Comment:
May be it make sense create one variable for the WebDriverWait and do not create new instance of it every time?